### PR TITLE
Upgrade zmon-worker

### PR DESCRIPTION
**Pull request by mohabusama**
Upgrade **zmon-worker**

Changes:

- Add alert evaluation timestamps for enhancing ZMON alerting & paging metrics
- Add ``scard`` and ``zcard`` to ``redis()`` wrapper
- Add ``query_batch`` to ``kairosdb()`` wrapper
- Fix ``cassandra`` wrapper connection leakage (including updated cassandra driver)
- Some enhancements to various helper functions: ``timeseries``, ``percentile``
- Add ``memcached`` wrapper

### CODE REVIEW

**Comment by mikkeloscar** [comment]
:+1:

File: cluster/manifests/zmon-worker/deployment.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ spec:
 
       containers:
         - name: zmon-worker
-          image: "pierone.stups.zalan.do/stups/zmon-worker:zv228"
+          image: "pierone.stups.zalan.do/stups/zmon-worker:v185-zv243"
```
Comment (linki) [inline cluster/manifests/zmon-worker/deployment.yaml]: @mohabusama does the image exist?

```console
$ pierone tags stups zmon-worker | grep v185-zv243 | wc -l
0
```

Comment (mohabusama) [reply cluster/manifests/zmon-worker/deployment.yaml]: Thanks for spotting this one. Will update!!

**Comment by linki** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**